code/renderers/svelte/src/docs/extractArgTypes.test.ts

```diff
@@ -87,7 +87,7 @@ describe('Extracting Arguments', () => {
           "control": Object {
             "type": "boolean",
           },
-          "description": null,
```

Comment:
@zhyd1997 — Can you please share why this change was necessary?

Reply:
I didn't change svelte code, and I think they 're all falsy value, so I just ran something like `update-snapshots` command.

Reply:
@kylegach 

Previously we didn't enable `code/renderers` unit test after moved all source code into `code` folder, and I need run unit test for `react`, so I manually enabled it in this PR, but `svelte` unit test failed, that's why I updated all `code/renders` snapshots here.